src/convert.rs

```diff
@@ -0,0 +1,139 @@
+/// Converts spiral coordinates to and from cube (q,r,s) coordinates
+use crate::position::{ring, ring_offset};
+
+/// Convert spiral hex coordinate x to cube coords (q,r,s).
+/// See: https://www.redblobgames.com/grids/hexagons/
+/// for a definition of cube coords.
+fn spiral_to_cube(x: usize) -> Result<(i8, i8, i8), &'static str> {
```

Comment:
```suggestion
fn spiral_to_cube(x: usize) -> (i8, i8, i8) {
```
since I don't see any errors being returned from this function